rest/src/main/java/com/confighub/api/repository/client/AClientAccessValidation.java

```diff
@@ -204,33 +200,15 @@ public void validatePull(@NotNull String contextString,
                              String remoteIp,
                              Store store,
                              Gson gson,
-                             String securityProfiles,
-                             boolean validateAuthOnly)
+                             String securityProfiles)
             throws ConfigException
     {
-        long start = System.currentTimeMillis();
-        resolved = ImmutableMap.of();
-        if (!validateAuthOnly)
-        {
-            com.confighub.core.resolver.Context context = resolveContext(contextString, store);
-            resolved = ImmutableMap.copyOf(context.resolveForClient());
-            log.info("Client [%s] from [%s] resolved %d keys in %d/ms > %s for repository [%s]",
+        com.confighub.core.resolver.Context context = resolveContext(contextString, store);
```

Comment:
this will now resolveContext for cached files, while i know we made some improvements, there i don't think we need to even do this function call if the cache is going to be found, ie we can avoid any db calls on cache hits.

Reply:
Ok i think resolveContext never hit the db, it was context.resolveForClient that does

Reply:
I can just remove that line, actually. It's there for validation, but:
1. If it's in the cache, it was already valid, and
2. If it's not in the cache, we call `resolveContext()` first thing anyway.